update_conda_lock/action.yml

```diff
@@ -0,0 +1,129 @@
+name: 'Conda Lock Updating Action'
+description: 'Action Pushes a Branch with Conda Lock Update'
+
+inputs:
+  commit_message:
+    description: 'Message of the bump commit'
+    required: true
+    default: '[BOT] Bump Conda Lock'
+  user_name:
+    description: 'Name of the user creating commit'
+    required: true
+    default: 'CONDA_LOCK_UPDATING_BOT'
+  user_email:
+    description: 'Email address of the user creating commit'
+    required: true
+    default: '<>'
```

Comment:
Instead of providing a dummy default, I suggest not providing a default.

Reply:
It seems to me that without providing any default value using this action in a workflow would always require passing this value. This dummy default simply serves the purpose of satisfying the command `git config user.email` if the workflow didn't pass that value to the Action.

Reply:
@ajelinski, I believe that's the purpose of `required`. If it is required, the content needs to be meaningful. If the content can be irrelevant, then it should be `required: false` and the Action should check it and provide a default internally.